### PR TITLE
Update regression tests

### PR DESCRIPTION
**Pull request by yifanz0**
## Checklist

- [ ] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [ ] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: src/Storage/RegressionTests/adls_setaclresusive.ps1 [hunk 1]
```diff
@@ -50,7 +50,7 @@ BeforeAll {
     $ctx2 = New-AzStorageContext  $testNode.accountName
 
     $filesystemName = "adlstest2"
-    $localSrcFile = ".\data\testfile_1024K_0" #The file needs to exist before tests, and should be 512 bytes aligned
```
Comment (blueww) [inline src/Storage/RegressionTests/adls_setaclresusive.ps1:53]: The line #46-47 also can't pass on my machine. 
Following can pass

$storageAccountName = $testNode.accountName
$storageAccountKey = (Get-AzStorageAccountKey -ResourceGroupName $resourceGroupName -Name $storageAccountName)[0].Value

**Review by blueww — APPROVED** [review]
Only 1 comments, others looks good to me.

File: src/Storage/RegressionTests/adls_setaclresusive.ps1 (hunk 1 above)
Comment (yifanz0) [reply src/Storage/RegressionTests/adls_setaclresusive.ps1:53]: It will work with the updated config file

Comment (yifanz0) [reply src/Storage/RegressionTests/adls_setaclresusive.ps1:53]: Hmm actually let me make the change as this info is actually redundant in the config

**Pull request merged**